org.jacoco.core/src/org/jacoco/core/internal/analysis/filter/KotlinNoSourceLinesFilter.java

```diff
@@ -0,0 +1,59 @@
+/*******************************************************************************
+ * Copyright (c) 2009, 2018 Mountainminds GmbH & Co. KG and Contributors
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Nikolay Krasko - initial API and implementation
+ *
+ *******************************************************************************/
+package org.jacoco.core.internal.analysis.filter;
+
+import org.objectweb.asm.tree.AbstractInsnNode;
+import org.objectweb.asm.tree.LineNumberNode;
+import org.objectweb.asm.tree.MethodNode;
+
+import java.util.ListIterator;
+import java.util.Set;
+
+public class KotlinNoSourceLinesFilter implements IFilter {
```

Comment:
JavaDoc missing.

Reply:
In addition to a missing Javadoc - maybe also rename into `KotlinGeneratedFilter` by analogy with existing `GroovyGeneratedFilter` and `LombokGeneratedFilter` ?